modules/ducktests/src/main/java/org/apache/ignite/internal/ducktest/tests/calcite/CalciteTestingApplication.java

```diff
@@ -0,0 +1,351 @@
+        tests.add(new QueryTest("SELECT JSON_DEPTH('{\"a\":1}') FROM t", 2));
+        tests.add(new QueryTest("SELECT JSON_KEYS('{\"a\":1}') FROM t", "[\"a\"]"));
+        tests.add(new QueryTest("SELECT JSON_PRETTY('{\"a\":1}') FROM t", "{\n  \"a\" : 1\n}"));
+        tests.add(new QueryTest("SELECT JSON_LENGTH('{\"a\":1}') FROM t", 1));
+        tests.add(new QueryTest("SELECT JSON_REMOVE('{\"a\":1, \"b\":2}', '$.a') FROM t", "{\"b\":2}"));
+        tests.add(new QueryTest("SELECT JSON_STORAGE_SIZE('1') FROM t", 1));
+        tests.add(new QueryTest("SELECT JSON_OBJECT('a': 1) FROM t", "{\"a\":1}"));
+        tests.add(new QueryTest("SELECT JSON_ARRAY('a', 'b') FROM t", "[\"a\",\"b\"]"));
+        tests.add(new QueryTest("SELECT '{\"a\":1}' IS JSON FROM t", true));
+        tests.add(new QueryTest("SELECT '{\"a\":1}' IS JSON VALUE FROM t", true));
+        tests.add(new QueryTest("SELECT '{\"a\":1}' IS JSON OBJECT FROM t", true));
+        tests.add(new QueryTest("SELECT '[1, 2]' IS JSON ARRAY FROM t", true));
+        tests.add(new QueryTest("SELECT '1' IS JSON SCALAR FROM t", true));
+        tests.add(new QueryTest("SELECT '{\"a\":1}' IS NOT JSON FROM t", false));
+        tests.add(new QueryTest("SELECT '{\"a\":1}' IS NOT JSON VALUE FROM t", false));
+        tests.add(new QueryTest("SELECT '{\"a\":1}' IS NOT JSON OBJECT FROM t", false));
+        tests.add(new QueryTest("SELECT '[1, 2]' IS NOT JSON ARRAY FROM t", false));
+        tests.add(new QueryTest("SELECT '1' IS NOT JSON SCALAR FROM t", false));
+
+        // XML
+        tests.add(new QueryTest("SELECT EXTRACTVALUE('<a>b</a>', '//a') FROM t", "b"));
+        tests.add(new QueryTest(
+            "SELECT XMLTRANSFORM('<a>b</a>','<?xml version=\"1.0\"?><xsl:stylesheet version=\"1.0\" xmlns:xsl=\"http://www.w3.org/1999/XSL/Transform\">  <xsl:output method=\"text\"/>  <xsl:template match=\"/\">    a - <xsl:value-of select=\"/a\"/>  </xsl:template></xsl:stylesheet>') FROM t",
+            "    a - b"
+        ));
+        tests.add(new QueryTest("SELECT \"EXTRACT\"('<a><b>c</b></a>', '/a/b') FROM t", "<b>c</b>"));
+        tests.add(new QueryTest("SELECT EXISTSNODE('<a><b>c</b></a>', '/a/b') FROM t", 1));
+
+        // Execute all tests
+        for (QueryTest test : tests) {
+            try (PreparedStatement stmt = conn.prepareStatement(test.qry);
+                 ResultSet rs = stmt.executeQuery()) {
+
+                List<Object> actualResults = new ArrayList<>();
+                ResultSetMetaData meta = rs.getMetaData();
+                int colCnt = meta.getColumnCount();
+
+                while (rs.next()) {
+                    for (int i = 1; i <= colCnt; i++)
+                        actualResults.add(rs.getObject(i));
+                }
+
+                if (!Objects.equals(test.expectedResults, actualResults)) {
+                    String errorMsg = String.format(
+                        "Query failed: %s\nExpected: %s\nActual: %s",
```

Comment:
Never use line breaks in error messages to be logged.  Logs are processed line by line.

Besides others it breaks the ducktest's approach to detect status of application run.  Just try the wrong expectedResult  for some testcase to see how it's handled and you will see.